runtime/doc/develop.txt

```diff
@@ -311,27 +516,29 @@ get_env_value()	Linux system function
 
 VARIOUS							*style-various*
 
-Typedef'ed names should end in "_T": >
-    typedef int some_T;
-Define'ed names should be uppercase: >
+Define'ed names should be uppercase:
+>
     #define SOME_THING
-Features always start with "FEAT_": >
+<
+
+Features always start with "FEAT_":
+>
     #define FEAT_FOO
+<
 
 Don't use '\"', some compilers can't handle it.  '"' works fine.
 
-Don't use:
-    #if HAVE_SOME
 Some compilers can't handle that and complain that "HAVE_SOME" is not defined.
 Use
     #ifdef HAVE_SOME
 or
     #if defined(HAVE_SOME)
-
+Don't use:
+    #if HAVE_SOME
```

Comment:
I think this made more sense in it’s original position (now the previous block’s “that” has no antecedent).